dags/glam.py

```diff
@@ -262,6 +267,33 @@
     arguments=('--append_table','--noreplace',),
     dag=dag)
 
+glam_client_probe_counts_extract = bigquery_etl_query(
+    task_id="glam_client_probe_counts_extract",
+    destination_table="glam_client_probe_counts_extract_v1",
+    dataset_id=dataset_id,
+    project_id="moz-fx-data-shared-prod",
+    owner="robhudson@mozilla.com",
+    email=["telemetry-alerts@mozilla.com", "msamuel@mozilla.com", "robhudson@mozilla.com"],
+    date_partition_parameter=None,
+    arguments=('--replace',),
+    dag=dag)
+
+glam_extract_clear_bucket = GoogleCloudStorageDeleteOperator(
+    task_id="glam_gcs_delete_old_extracts",
```

Comment:
I think maybe `task_id` needs to match the variable for `GoogleCloudStorageDeleteOperator`.

So `task_id` would also be `glam_extract_clear_bucket` or vice versa.